test/test_message_converter.py

```diff
@@ -286,6 +286,22 @@ def test_dictionary_with_empty(self):
         expected_message = serialize_deserialize(expected_message)
         self.assertEqual(message, expected_message)
 
+    def test_dictionary_with_empty_additional_args_strict_mode(self):
+        from std_msgs.msg import Empty
+        dictionary = {"additional_args": "should raise value error"}
+        with self.assertRaises(ValueError) as context:
+            message_converter.convert_dictionary_to_ros_message('std_msgs/Empty', dictionary)
+        print(context.exception.message.find("has no field named") >= 0)
```

Comment:
The print statement should be removed.

Reply:
leftover from getting the test to run --> removed and changed the commit